.github/workflows/build-client-packages.yml

```diff
@@ -0,0 +1,126 @@
+name: Build client packages
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
```

Comment:
@bedroge Should we add a section in the documentation that tags have to start with `v` to ensure that a release is made with the packages as assets?

Reply:
Yeah, where shall we add that?

Reply:
In the README.md file of this repo, new subsection?

Reply:
Will do that tomorrow.

Reply:
Done! Could you review this?